bats/tests/helpers/utils.bats

```diff
@@ -530,3 +617,35 @@ get_json_test_data() {
     assert_success
     assert_output "${COUNTER}_3.png"
 }
+
+########################################################################
+
+@test 'save_var existing variables' {
+    FOO=baz
+    BAR=foo
+    save_var FOO BAR
+}
+
+@test 'load_var existing variables' {
+    # shellcheck disable=SC2030
+    FOO=bar BAR=bar
+    load_var FOO BAR
+    [[ $FOO == baz ]]
+    [[ $BAR == foo ]]
+}
+
+@test 'save_var of non-existing variable' {
+    run save_var DOES_NOT_EXIST
```

Comment:
Just realized that `save_var` will abort on the first variable that is unset. For symmetry with `load_var` it should probably continue to try to save any remaining variables.